tests/FSharp.AWS.DynamoDB.Tests/MetricsCollectorTests.fs

```diff
@@ -139,94 +137,94 @@ type ``Metrics Collector Tests`` (fixture : TableFixture) =
         let gsk = guid()
-        let items = [| for i in 0 .. 1000 -> mkItem hk gsk i |]
+        let items = [| for i in 0 .. 24 -> mkItem hk gsk i |]
         for item in items do
           table.PutItem item |> ignore
 
         let collector = TestCollector()
         let sut = table.WithMetricsCollector(collector.Collect)
 
-        let items = sut.Scan()
+        let _ = sut.BatchPutItems(items |> Seq.map (fun i -> { i with LocalAttribute = 1000 }))
 
         test <@ match collector.Metrics with
-                | [{ ItemCount = c; Operation = Scan; TableName = ExpectedTableName; ConsumedCapacity = TotalCu cu }] ->
+                | [{ ItemCount = c; Operation = BatchWriteItems; TableName = ExpectedTableName; ConsumedCapacity = TotalCu cu }] ->
                     c = items.Length
                     && cu > 0
                 | _ -> false @>
 
-    let [<Fact>] ``Collect Metrics on Query`` () =
+    let [<Fact>] ``Collect Metrics on BatchDeleteItem`` () =
         let hk = guid()
         let gsk = guid()
-        let items = [| for i in 0 .. 1000 -> mkItem hk gsk i |]
+        let items = [| for i in 0 .. 24 -> mkItem hk gsk i |]
         for item in items do
           table.PutItem item |> ignore
 
         let collector = TestCollector()
         let sut = table.WithMetricsCollector(collector.Collect)
 
-        let items = sut.Query(<@ fun (r : MetricsRecord) -> r.HashKey = hk @>)
+        let _ = sut.BatchDeleteItems(items |> Seq.map (fun i -> TableKey.Combined (i.HashKey, i.RangeKey)))
 
         test <@ match collector.Metrics with
-                | [{ ItemCount = c; Operation = Query; TableName = ExpectedTableName; ConsumedCapacity = TotalCu cu }] ->
+                | [{ ItemCount = c; Operation = BatchWriteItems; TableName = ExpectedTableName; ConsumedCapacity = TotalCu cu }] ->
                     c = items.Length
                     && cu > 0
                 | _ -> false @>
 
-    let [<Fact>] ``Collect Metrics on BatchGetItem`` () =
-        let hk = guid()
-        let gsk = guid()
+    interface IClassFixture<TableFixture>
+
+/// TableFixture with 1000 items with a known HashKey pre-inserted
+type ManyItemsFixture() =
+    inherit TableFixture()
+
+    // TOCONSIDER shift this into IAsyncLifetime.InitializeAsync
+    let table = TableContext.Create<MetricsRecord>(base.Client, base.TableName, createIfNotExists = true)
+
+    let hk = guid ()
+    do  let gsk = guid ()
         let items = [| for i in 0 .. 99 -> mkItem hk gsk i |]
         for item in items do
-          table.PutItem item |> ignore
+            table.PutItem item |> ignore
 
-        let collector = TestCollector()
-        let sut = table.WithMetricsCollector(collector.Collect)
+    member _.Table = table
+    member _.HashKey = hk
+
+/// NOTE These tests share the prep work of making a Table Containing lots of items to read
+// DO NOT add tests that will delete or mutate those items
+type ``Bulk Read Operations``(fixture : ManyItemsFixture) =
```

Comment:
note these are moved to a separate Test Class so they can

1. share the work of establishing the items
2. run in parallel with the other tests